Aztec/Classes/Public/TextKit/TextAttachment.swift

```diff
@@ -120,11 +139,37 @@ public class TextAttachment: NSTextAttachment
         let containerWidth = imageBounds.size.width
         let origin = CGPoint(x: xPosition(forContainerWidth: imageBounds.size.width), y: 0)
         let size = CGSize(width: onScreenWidth(containerWidth), height: onScreenHeight(containerWidth))
-        let scale = UIScreen.mainScreen().scale
 
-        UIGraphicsBeginImageContextWithOptions(imageBounds.size, false, scale)
+        UIGraphicsBeginImageContextWithOptions(imageBounds.size, false, 0)
```

Comment:
Can we wrap the drawing code in a separate method?  Feels like it'd make this method a bit easier to read.

Something like `let glyphImage = imageForEtc...()`